app/developers/forms.py

```diff
@@ -49,5 +51,13 @@ class ConditionSelectQuestionForm(FlaskForm):
     )
     submit = SubmitField("Continue", widget=GovSubmitInput())
 
-    def add_question_options(self, questions: list["Question"]) -> None:
-        self.question.choices = [(question.id, f"{question.text} ({question.name})") for question in questions]
+    def __init__(self, *args, question: "Question", **kwargs):  # type: ignore[no-untyped-def]
+        super().__init__(*args, **kwargs)
+
+        supported_form_questions = [
```

Comment:
It looks to me like this is what would actually consume the (nicely test covered) `get_supported_form_questions` but it seems to re-implement that logic here. (I may have missed something here!)